text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+upgrade between Rust stable releases.
+
+However, for various institutional reasons, many users do not find it
+convenient and desirable to upgrade with the frequency that Rust releases. So
+far, they have been left on their own to figure out when to actually upgrade,
+creating a situation of uncertainty and disparity. Moreover, because there are
+no norms around when an older stable stops being "supported," libraries authors
+experience a similar level of uncertainty around which versions of the compiler
+their library should be compatible with.
+
+We solve this problem by adding long-term support ("LTS") release channels,
+providing explicit support for releases beyond the latest stable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Two new channels: lts and lts-prior
+
+In addition to `stable`, `beta`, and `nightly`, we will add two additional
+release tracs:
+
+* `lts`: The most recent LTS release.
+* `lts-prior`: The second most recent LTS release.
+
+Every fourth stable/beta release, we will perform a release on both the `lts`
+and `lts-prior` channels. The **previous** stable release will be the basis for
+the new `lts` release, and the previous `lts` release will move to the
+`lts-prior` release. This is analogous to the way that the previous beta
+becomes the new stable.
+
+To keep it as easy as possible to remember which releases are lts release, we
+will begin the lts schedule so that the point version of the LTS release is
+always divisible by four. Depending on implementation schedule, that means that
+the first LTS release will probably be 1.28 or 1.32, release concurrently with
+either 1.29 or 1.33.
+
+## Level of support from the Rust project
+
+All three of the stable release channels - stable, lts, and lts-prior - will
+receive the same level of support. We will host documentation for all three
+channels and provide equivalent levels of patch release support - that is, if a
+bug justifying a patch exists in any of these channels, we will release a patch
+for that channel, just as we do today for stable. However, lts releases are
+less likely to receive patch releases in practice, since these bugs are most
+often found in the first six weeks after a stable release.
+
+As a result of this, a stable release which is a candidate for the lts channel
+(that is, its minor version is a multiple of four) will be supported across all
+three channels for the following nine release cycles, a period of 54 weeks or
+just over a year.
+
+## Recommendations for library authors
+
+Today, library authors fall into three general camps:
+
+1. Their library compiles on the nightly channel.
+2. Their library compiles on the stable channel.
+3. Their library compiles on any release more recent than an arbitrary old
+release, which is different for each library (for example, `failure` compiles
+on 1.18.0, whereas `serde` compiles on 1.13.0).
```

Comment:
Mostly unrelated, but worth mentioning -- Serde only supports 1.13 with default features turned off. With them enabled, it only supports 1.24 and later -- Diesel supports specifically that version since one of our dependencies requires serde and didn't set `default-features = false`. While not specifically related to this RFC, it's a larger problem that is tangentially related to the one this seems to be trying to solve.